src/components/legacy/markdown/youtube-embed.tsx

```diff
@@ -1,96 +1,55 @@
-import React, { useContext } from 'react';
+import React from 'react';
 import styled from 'styled-components';
-import { theme } from '../../layout/theme';
-import { Button } from '../../ui/buttons/button';
-import { TextStyles } from '../../typography';
-import YouTubeConsentContext from '../../../context/youtube-consent-context';
-import { Link } from '../links/links';
 
 interface YoutubeEmbedProps {
   videoId: string;
 }
 
-// how to make iframe responsive: https://stackoverflow.com/questions/17838607/making-an-iframe-responsive
-const StyledIframe = styled.iframe`
-  position: absolute;
-  top: 0;
-  left: 0;
-  width: 100%;
-  height: 100%;
-`;
-
 const YoutubeEmbedWrapper = styled.div`
   position: relative;
   padding-bottom: 56.25%; /* 16:9 */
   padding-top: 25px;
   height: 0;
-`;
-
-const YouTubePrivacyBannerWrapper = styled.div`
-  position: absolute;
-  top: 0;
-  left: 0;
-  width: 100%;
-  height: 100%;
-  padding: 16px;
-  gap: 16px;
-  background-color: ${theme.palette.background.card};
-  display: flex;
-  flex-direction: column;
-  justify-content: center;
-  align-items: center;
-`;
-
-const LegalText = styled.p`
-  ${TextStyles.label}
-  text-align: center;
-`;
 
-const StyledButton = styled(Button)`
-  width: fit-content;
-`;
+  --im-font-family: 'CocoGothic', sans-serif;
 
-const SyledLink = styled(Link)`
-  color: ${theme.palette.text.link.default};
+  div[data-service] .cll .c-n-a {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-wrap: wrap;
+  }
 
-  &:hover {
-    border-bottom: 1px solid ${theme.palette.text.link.default};
+  div[data-service] .cll .c-l-b,
+  div[data-service] .cll .c-la-b {
+    display: inline-flex;
+    align-items: center;
+    padding: 8px 12px;
+    color: #ffffff;
+    border-radius: 30px;
+    cursor: pointer;
+    background: linear-gradient(275.41deg, #543fd7 0%, #2756fd 100%);
+    text-wrap: nowrap;
+    width: fit-content;
+
+    @media (max-width: 768px) {
+      margin-left: 0;
+      margin-bottom: 8px;
+    }
```

Comment:
Please recheck the button layout with the existing buttons from other pages. The height is not quite accurate yet, I suspect it is due to the font style, but I have not checked it more closely.

Also, here's a suggestion for refactoring: The `gap` property is quite suitable for this use case. This allows you to remove the media query and, in addition, there is still a gap between the buttons on large mobile devices


```diff
diff --git a/src/components/legacy/markdown/youtube-embed.tsx b/src/components/legacy/markdown/youtube-embed.tsx
index d10ce1d..3e74397 100644
--- a/src/components/legacy/markdown/youtube-embed.tsx
+++ b/src/components/legacy/markdown/youtube-embed.tsx
@@ -18,6 +18,7 @@ const YoutubeEmbedWrapper = styled.div`
     justify-content: center;
     align-items: center;
     flex-wrap: wrap;
+    gap: 8px;
   }
 
   div[data-service] .cll .c-l-b,
@@ -31,11 +32,7 @@ const YoutubeEmbedWrapper = styled.div`
     background: linear-gradient(275.41deg, #543fd7 0%, #2756fd 100%);
     text-wrap: nowrap;
     width: fit-content;
-
-    @media (max-width: 768px) {
-      margin-left: 0;
-      margin-bottom: 8px;
-    }
+    margin: 0;
   }
 `;
```